sdk/core/Azure.Core/src/Pipeline/Policies/RetryPolicy.cs

```diff
@@ -10,16 +10,49 @@
 
 namespace Azure.Core.Pipeline.Policies
 {
-    public abstract class RetryPolicy : HttpPipelinePolicy
+    public class RetryOptions
     {
-        private const string RetryAfterHeaderName = "Retry-After";
-        private const string RetryAfterMsHeaderName = "retry-after-ms";
-        private const string XRetryAfterMsHeaderName = "x-ms-retry-after-ms";
-
         /// <summary>
         /// Gets or sets the maximum number of retry attempts before giving up.
         /// </summary>
         public int MaxRetries { get; set; } = 10;
+    }
+
+    public class ExponentialRetryOptions: RetryOptions
```

Comment:
I'm guessing "because prototype" but, my are there many classes in this here file.

Reply:
Done. I also removed the base type.